devops/arangodb.yml

```
@@ -111,3 +95,19 @@
         name: arangodb3
         state: restarted
       become: true
+
+    - name: Template file to server
+      template:
+        src: create-readonly-user.js
+        dest: /tmp/create-readonly-user.js
+        owner: ubuntu
+        mode: 444
+
+    - name: Ensure there is a read-only user
+      command: arangosh --server.password {{ arangodb_root_password }} --server.endpoint ssl://127.0.0.1:8529 --javascript.execute /tmp/create-readonly-user.js
+      become: true
+
+    - name: Delete templated file
```

Same comment as above. Maybe:

```suggestion
    - name: Delete user provisioning script
```